.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ workflows:
           name: test_with_go_<< matrix.go_version >>
           matrix: &go-version-matrix
             parameters:
-              go_version: ["1.14.7", "1.15.15"]
+              go_version: ["1.16.11"]
```

Comment:
if you're replacing the two we used to test against with one, might as well test the two supported releases (Go 1.16.11 and go 1.17.4)

Reply:
added, thanks